packages/frontend-v2/pages/home.tsx

```diff
@@ -65,13 +77,14 @@ const HomePage: NextPage = () => {
     logger.error("HomePage", error);
     handleApiClientError(error, router);
 
-    // render just the boiler plate page since we couldn't fetch the student w plans
-    return <PageLayout />;
+    // If we couldn't fetch the student's plan, show a blank page for now.
+    // We might want to show some more actionable error in the future.
+    return <div></div>;
   }
 
   // handle loading state
   if (!student) {
-    return <LoadingPage pageLayout={PageLayout} />;
```

Comment:
Previously we loaded the entire page as a skeleton UI of sorts, this had the unintended side-effect of causing the Sidebar to mount multiple times, resulting in us sending twice as many queries to Search's API. Without the skeleton UI, the page shows a loading spinner for the <1s that the page takes to load in.